website/pages/docs/plugin-portal/index.mdx

```diff
@@ -101,6 +101,10 @@ reviewed by HashiCorp.
 
 - [Venafi](https://github.com/Venafi/vault-pki-backend-venafi)
 
+### Database
+
+- [Aerospike](https://github.com/aerospike-community/vault-plugin-database-aerospike)
+
```

Comment:
Can this be moved up to be above `### Secrets` to keep it consistent with the "Official" layout.

Reply:
Yes.  That makes sense.

Reply:
I can swap the sub-sections to keep them in consistent order.

Reply:
@calvn @acahn Done.

Reply:
LGTM!  Thank you Ronen